app/src/ui/lib/commit-attribution.tsx

```diff
@@ -40,23 +40,15 @@ export class CommitAttribution extends React.Component<
         <span className="authors">{this.renderAuthorInline(authors[0])}</span>
       )
     } else if (authors.length === 2) {
-      const title = authors.map(a => a.name).join(', ')
-
       return (
-        <span className="authors" title={title}>
```

Comment:
Removed this in anticipation of the commit expansion work.
![image](https://github.com/desktop/desktop/assets/75402236/7a497c30-d469-475b-87e4-d02316d9065d)